web/src/components/CodeSample/CodeSample.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react'
+import hljs from 'highlight.js'
+import hljsDefineGraphQL from 'highlightjs-graphql'
+hljsDefineGraphQL(hljs)
+const md = require('markdown-it')({
```

Comment:
I'm a big fan of remark, any reason you used markdown-it?

Reply:
Markdown-it is used on redwoodjs.com so I actually just copied all the settings for it. Not opposed to switching to remark, just tried to maintain consistency

Reply:
Thought so. Consistency is good. Keep it 👍 
Unless @dthyresson has a strong opinion

Reply:
Nope! Let's keep for now unless there's a feature in remark we need.